contentcuration/contentcuration/urls.py

```diff
@@ -350,7 +352,8 @@ def get_queryset(self):
     urlpatterns += [
         url(r'^' + settings.STORAGE_URL[1:] + '(?P<path>.*)$', file_views.debug_serve_file, name='debug_serve_file'),
         url(r'^' + settings.CONTENT_DATABASE_URL[1:] + '(?P<path>.*)$', file_views.debug_serve_content_database_file, name='content_database_debug_serve_file'),
-        url(r'^' + settings.CSV_URL[1:] + '(?P<path>.*)$', file_views.debug_serve_file, name='csv_debug_serve_file')
+        url(r'^' + settings.CSV_URL[1:] + '(?P<path>.*)$', file_views.debug_serve_file, name='csv_debug_serve_file'),  # this seems broken since CSVs dont use content storage
```

Comment:
Currently (in develop) serving CSVs is not working because `debug_serve_file` assumes we're serving only content-addressable files under `/storage/content/`.

Placing a sample file in `contentcuration/csvs/a.csv` and trying to access it at `/content/csvs/a.csv` raises a `string index out of range` ERROR. For CSVs this is probably not necessary --- they just need to be there so that django can send the email, but for `/content/jsons` we'll need some sort of static serve.